### PR TITLE
Add slack page to the website

### PR DESCRIPTION
**Pull request by ghickman**
This adds the slack rules page.

### CODE REVIEW

File: pages/slack.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title: Slack
+---
+
+# Rules for our Slack channel
+As with any of our conference spaces (online or physical), participants are expected to follow our [Code of Conduct](/code-of-conduct/).
+
+In order to make the experience pleasant for everyone, we also ask that you follow some slack-specific rules:
+
+1. Do not message people privately without their consent, and do not invite them to join any room without their consent.
+2. Do not mention people who don't belong to particular room in that room.
```
Comment (alexwlchan) [inline pages/slack.md]: "to _a_ particular room"?

File: pages/slack.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title: Slack
+---
+
+# Rules for our Slack channel
+As with any of our conference spaces (online or physical), participants are expected to follow our [Code of Conduct](/code-of-conduct/).
+
+In order to make the experience pleasant for everyone, we also ask that you follow some slack-specific rules:
```
Comment (alexwlchan) [inline pages/slack.md]: Nitpick: "slack" should be capitalised.

File: pages/slack.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+title: Slack
+---
+
```
Comment (inglesp) [inline pages/slack.md:5]: Could we have an introductory section here?  How about:

"This year we're experimenting with a conference [Slack](https://slack.com/), for conference delegates to communicate with each other and with the organisers during the conference.

All delegates should receive an email invitation to join the team.  [Get in touch](/contact/) if you think you should have received one but haven't.

We plan to close the team on November 19th, two months after the end of the conference."

File: pages/slack.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+# Rules for our Slack channel
+As with any of our conference spaces (online or physical), participants are expected to follow our [Code of Conduct](/code-of-conduct/).
+
+In order to make the experience pleasant for everyone, we also ask that you follow some Slack-specific rules:
+
```
Comment (inglesp) [inline pages/slack.md:17]: Could we have a rule about being respectful of people's time in DMs?

File: pages/slack.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+1. Do not mention people who don't belong to a particular room in that room.
+2. Please keep job postings to the #jobs channel.
```
Comment (inglesp) [inline pages/slack.md]: How about "Try to keep on-topic in each channel, and only post job postings to the #jobs channel."?  I think this should actually be rule 1.

File: pages/slack.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+3. Do not edit a channel's topic without the organizers' approval.
+4. We've deliberately restricted the creations on new channels but we're happy to create some upon request so feel free to get in touch if you have an idea for a channel that you think would be useful.
```
Comment (inglesp) [inline pages/slack.md]: This isn't really a rule.  Could we move it to the introduction?

**Pull request merged**